### ISSUE TITLE
[loqui im, whats app, firefox] How to give forever access to the microphone on firefox os

### ISSUE THREAD

**Issue by muradaliev** [loqui/im#1218]
I do not can send audio message because it request acces every time when i click on the micrifone icon.
Sorry for my bad english, i hope your understand me)  Big Thanks

**Comment by nfsprodriver** [comment]
Good point! This is a known issue, which persist since a new FFOS version (1.3 / 2.0 ?) On UT this doesn't work also. So the whole voice message implementation seems to be useless and needs a rework. So I recommend you to record your voice in another app and upload the record as a media file.

**Comment by bryanjhv** [comment]
Firefox OS 1.3 suffers this limitation also. Dictaphone app solves this partially by calling `getUserMedia` and [caching the stream](https://github.com/laaglu/dictaphone.js/blob/1be8888c859885953465bcb3346a1ede518dd479/app/AudioEnv.js#L28-L54) for later reuse. It's a bit weird for the user to have "recording" icon always on notifications bar, but that's the only way that will work.

**Comment by Honk2** [comment]
This is not an issue on FFOS 2.0
I granted access and it is working since then.
Also I successfully can withdraw permission using the application manager. Please check there.
